api/Navigator.json

```diff
@@ -1025,10 +1025,12 @@
               "notes": "An outdated constraint syntax is still in use, but the syntax described here is available through the <a href='https://github.com/webrtc/adapter'>adapter.js</a> polyfill."
             },
             "safari": {
-              "version_added": false
+              "version_added": "11",
```

Comment:
Interesting. Confirmed with https://mdn-bcd-collector.appspot.com/tests/api/Navigator/getUserMedia that it's there in Safari 11.1.2, but not in 12.1.2.

I guess @youennf had a change of heart :)